src/commands/backend/metrics.ts

```diff
@@ -0,0 +1,53 @@
+import { Command } from '@oclif/core'
+import { Jamsocket } from '../../jamsocket'
+import chalk from 'chalk'
+
+function prettifyBytes(bytes: number) {
+  const suffixes = ['B', 'KiB', 'MiB', 'GiB', 'TiB']
+  const i = Math.trunc(Math.log2(bytes) / 10)
+  return (bytes / (1 << (i * 10))).toFixed(2) + ' ' + suffixes[i]
+}
+
+type BackendMetrics = {
+  cluster: string,
+  backend_id: string,
+  mem_used: number,
+  mem_available: number,
+  cpu_used: number,
+  sys_cpu: number
+}
+
+const resetLine = '\r\u001B[0J'
```

Comment:
From offline convo: can we add a comment here saying that this clears from the line the cursor's on to the bottom of the screen?